prepare/templates/qa/with_context.py

```diff
@@ -63,6 +64,32 @@
     overwrite=True,
 )
 
+add_to_catalog(
+    MultiReferenceTemplate(
+        instruction="Answer the question based on the information provided in the {context_type} given below.",
```

Comment:
```suggestion
        instruction="Answer the question based on the information provided in the given {context_type}.",
```
"below" assume something about the spatial order (Format Role)